manager/manager.go

```diff
@@ -959,6 +963,16 @@ func (m *manager) createContainerLocked(containerName string, watchSource watche
 				klog.Infof("perf_event metrics will not be available for container %s: %s", cont.info.Name, err)
 			}
 		}
+
+		cpuCgroupPath, err := handler.GetCgroupPath("cpu")
+		if err != nil {
+			klog.Warningf("Error getting cpu cgroup path: %q", err)
+		} else {
+			cont.perfCollector, err = m.wssManager.GetCollector(cpuCgroupPath)
```

Comment:
Do not kill perf, please ;)

Reply:
Aaaand - what about other storages? I guess we should implement them too.

Reply:
I'd like to add support for different storages in separate PR